### PR TITLE
HPCC-20747 Bloating ESP log with SSL messages

### PR DESCRIPTION
**Pull request by roscoche**
## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [X] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [X] My code follows the code style of this project.
  - [X] My code does not create any new warnings from compiler, build system, or lint.
- [X] The commit message is properly formatted and free of typos.
  - [X] The commit message title makes sense in a changelog, by itself.
  - [X] The commit is signed.
- [X] My change requires a change to the documentation.
  - [X] I have updated the documentation accordingly, or...
  - [X] I have created a JIRA ticket to update the documentation.
  - [X] Any new interfaces or exported functions are appropriately commented.
- [X] I have read the CONTRIBUTORS document.
- [X] The change has been fully tested:
  - [X] I have added tests to cover my changes.
  - [X] All new and existing tests passed.
  - [X] I have checked that this change does not introduce memory leaks.
  - [X] I have used Valgrind or similar tools to check for potential issues.
- [X] I have given due consideration to all of the following potential concerns:
  - [X] Scalability
  - [X] Performance
  - [X] Security
  - [X] Thread-safety
  - [X] Premature optimization
  - [X] Existing deployed queries will not be broken
  - [X] This change fixes the problem, not just the symptom
  - [X] The target branch of this pull request is appropriate for such a change.
- [X] There are no similar instances of the same problem that should be addressed
  - [X] I have addressed them here
  - [X] I have raised JIRA issues to address them separately
- [X] This is a user interface / front-end modification
  - [X] I have tested my changes in multiple modern browsers
  - [X] The component(s) render as expected

### CODE REVIEW

**Comment by richardkchapman** [comment]
Looks very similar to the changes recently checked into 7.0.0 branch by @gaojx 

I'm happy to merge this into 6.4.x stream, but it will need to wait for 6.2.30 as 6.4.28 is past final release status.

File: esp/bindings/http/platform/httpprot.cpp [hunk 1]
```diff
@@ -426,16 +426,16 @@ bool CHttpThread::onRequest()
...
         int res = 0;
         try
         {
-            DBGLOG("Accepting from secure socket");
+            ESPLOG(LogMax, "Accepting from secure socket");
             res = secure_sock->secure_accept();
             if(res < 0)
             {
-                DBGLOG("Error accepting from secure socket");
+                ESPLOG(LogMax, "Error accepting from secure socket");
```
Comment (rpastrana) [inline esp/bindings/http/platform/httpprot.cpp]: This should have a lower log level... if an error occurs, it is imperative we report it.

**Review by rpastrana — CHANGES_REQUESTED** [review]
@roscoche one comment...

File: esp/bindings/http/platform/httpprot.cpp (hunk 1 above)
Comment (roscoche) [reply esp/bindings/http/platform/httpprot.cpp]: @rpastrana I agree, thanks for noticing! Changed it to LogMin.

**Review by rpastrana — APPROVED** [review]
@roscoche looks good.

**Pull request merged**